beta/src/components/MDX/Sandpack/index.tsx

```diff
@@ -133,16 +135,25 @@ function Sandpack(props: SandpackProps) {
   }
 
   return (
-    <div className="my-8" translate="no">
+    <div className="sandpack-container my-8" translate="no">
       <SandpackProvider
-        key={key}
         template="react"
         customSetup={{...setup, files: files}}
-        autorun={autorun}>
+        autorun={autorun}
+        key={key}>
         <CustomPreset
           isSingleFile={isSingleFile}
+          showDevTools={showDevTools}
+          onDevToolsLoad={() => setDevToolsLoaded(true)}
+          devToolsLoaded={devToolsLoaded}
           onReset={() => {
-            setResetKey((k) => k + 1);
+            /**
```

Comment:
Hmm actually this doesn't seem necessary anymore? If I remove this onReset and even the key generation logic, it still works in development and replaces the sandbox when MDX content is edited. So we can remove this after all?